tasks/task01/narcissistic_numbers.py

```diff
@@ -0,0 +1,21 @@
+k = 1
+
+
+def get_digits_number(num: int) -> int:
+    count = 0
+    while num > 0:
+        count += 1
+        num = num // 10
+    return count
+
+
+def is_narcissistic_num(num: int) -> bool:
+    n: int = get_digits_number(num)
+    nth: [int] = [int(digit) ** n for digit in str(num)]
+    return num == sum(nth)

```

Comment:
You are summing up the list. And the list is not used anywhere else. 
To save memory you could create a generator object and pass it to the sum function. 
Check here
https://www.geeksforgeeks.org/python-list-comprehensions-vs-generator-expressions/#:~:text=The%20generator%20yields%20one%20item,memory%20efficient%20than%20the%20lists

Reply:
Solution was reworked